### PR TITLE
Add new page: Telegram (closes #44)

### LINKED ISSUE #44: Write new page to document RITlug Telegram
We need a new page to document the RITlug Telegram group, even though it is officially deprecated and no longer promoted. So long as it exists and we're using it, we need to make a mention of it somewhere.

### PR DESCRIPTION
**Pull request by jwflory**
This PR adds a new page to document the Telegram group in these ways:

* Background
* Administrator action
* Administrator list

A screenshot of the rendered documentation is below:

"Telegram" Runbook page rendering

### CODE REVIEW

File: docs/infrastructure/telegram.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Background
+**********
+
+Today, the Telegram group is left online as a convenience to older members who choose to use it.
+However, it is not officially advertised or endorsed by RITlug eboard.
+
+Before switching to Slack, RITlug advertised a Telegram group and :doc:`IRC channel <irc-channel>`.
+After switching to Slack and setting up a bridge, RITlug only advertised the Slack.
+However, the Telegram group and IRC channel were kept online for convenience.
+The work to maintain a bridge across all groups was simple: run a bot and forget.
```
Comment (jrtechs) [inline docs/infrastructure/telegram.rst]: I think we should link/explain how we are running the telegram bridge bot on this page.

**Comment by jwflory** [comment]
@jrtechs I pushed 21bc9ac2737899779f59f18a749fd2f77699cff7 to address your feedback:

Rendered preview of added Teleirc section

I think the Teleirc documentation belongs separately since it is another service. But mentioning it here is a good idea, and this will also make it obvious we need to write new documentation for Teleirc.

What do you think?

**Comment by jwflory** [comment]
@Tjzabel It is already documented in the Runbook (see [Other on-boarding tasks](http://runbook.ritlug.com/en/latest/administration/eboard-onboarding-offboarding.html#other-on-boarding-tasks)).

**Comment by Tjzabel** [comment]
My bad, didn't see it there!

**Pull request merged**